lib/utils/visUtil.hpp

```diff
@@ -527,6 +528,110 @@ class movingAverage {
     bool initialised = false;
 };
 
+/**
+ * @class SlidingWindowMinMax
+ *
+ * @brief Use two deques to keep tracking minimum and maximum values.
+ **/
+class SlidingWindowMinMax {
+
+public:
+    /**
+     * @brief Get the current minimum value from the front of minDeque.
+     *
+     * @return The current minimum value.
+     **/
+    double getMinimum();
```

Comment:
Since this isn't being included as an external library but instead being modified to become a part of our main source code, it should be formatted to match our coding standards. 
```suggestion
    double get_minimum();
```